depthai_sdk/setup.py

```diff
@@ -1,44 +1,51 @@
 import io
+from setuptools import setup, find_packages
 
-from setuptools import setup
-
+# Read requirements from requirements.txt
 with open('requirements.txt') as f:
     requirements = f.readlines()
 
-install_requires = [requirement for requirement in requirements if '--' not in requirement]
+# Flatten the install_requires list
+install_requires = [requirement.strip() for requirement in requirements if '--' not in requirement]
 
 setup(
     name='depthai-sdk',
     version='1.15.0',
     description='This package provides an abstraction of the DepthAI API library.',
-    long_description=io.open("README.md", encoding="utf-8").read(),
+    long_description=open('README.md').read(),
     long_description_content_type="text/markdown",
-    url='https://github.com/luxonis/depthai/tree/main/depthai_sdk',
+    url='https://github.com/dbrown99c/depthai/tree/main/depthai_sdk',
     keywords="depthai sdk oak camera",
     author='Luxonis',
     author_email='support@luxonis.com',
     license='MIT',
-    packages=['depthai_sdk', 'depthai_sdk_console_scripts'],
-    package_dir={'': 'src'},  # https://stackoverflow.com/a/67238346/5494277
+    packages=find_packages(where='src'),
+    package_dir={'': 'src'},
     install_requires=install_requires,
     include_package_data=True,
     extras_require={
-        "visualize": ['PySide2',
-                      'Qt.py>=1.3.0',
-                      'matplotlib==3.5.3; python_version <= "3.7"',
-                      'matplotlib==3.6.1; python_version > "3.7"',
-                      'depthai-viewer',
-                      'open3d'
-                      ],
-        "replay": ['mcap>=0.0.10',
-                   'mcap-ros1-support==0.0.8',
-                   'rosbags==0.9.11'],
-        "record": ['av'],
-        "test": ['pytest']
+        "visualize": [
+            'PySide2',
+            'Qt.py>=1.3.0',
+            'matplotlib==3.5.3; python_version <= "3.7"',
+            'matplotlib==3.6.1; python_version > "3.7"',
+            'depthai-viewer',
+            'open3d'
+        ],
+        "replay": [
+            'mcap>=0.0.10',
+            'mcap-ros1-support==0.0.8',
+            'rosbags==0.9.11'
+        ],
+        "record": [
+            'av'
+        ],
+        "test": [
+            'pytest'
+        ]
     },
     project_urls={
-        "Bug Tracker": "https://github.com/luxonis/depthai/issues",
-        "Source Code": "https://github.com/luxonis/depthai/tree/main/depthai_sdk",
+        "Source Code": "https://github.com/dbrown99c/depthai",
```

Comment:
This as well

Reply:
removed all the extraneous commits apologies thank you